ui/src/lib/gql/project/hook.ts

```diff
@@ -0,0 +1,92 @@
+import { useMutation, useQuery, useQueryClient } from "@tanstack/react-query";
+
+import { useGraphQLContext } from "@flow/lib/gql";
+import { CreateProject, DeleteProject, GetProjects, Project } from "@flow/types";
+
+import { CreateProjectInput, DeleteProjectInput } from "../__gen__/graphql";
+
+export enum ProjectQueryKeys {
+  GetProjects = "getProjects",
+}
+
+export const useProject = () => {
+  const graphQLContext = useGraphQLContext();
+  const queryClient = useQueryClient();
+
+  const createProjectMutation = useMutation({
+    mutationFn: async (input: CreateProjectInput) => {
+      const data = await graphQLContext?.CreateProject({ input });
+      return { project: data?.createProject?.project };
```

Comment:
Sorry realized you are passing the gql project here

Reply:
![image](https://github.com/reearth/reearth-flow/assets/20891087/6eaad631-d0a8-4e03-b71d-f21d62de5779)
I am using the one from types!